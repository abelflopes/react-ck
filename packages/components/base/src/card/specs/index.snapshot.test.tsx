import React from "react";
import { Card } from "../index";
import renderer from "react-test-renderer";

describe("snapshot Card", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Card>Card</Card>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
