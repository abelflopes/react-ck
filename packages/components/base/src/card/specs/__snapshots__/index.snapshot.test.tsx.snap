// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`snapshot Card renders correctly 1`] = `
<DocumentFragment>
  <div
    class="root bordered vertical spacing_m border_radius_m"
  >
    <div
      class="content"
    >
      Card
    </div>
  </div>
</DocumentFragment>
`;
